INSTALL

```diff
@@ -668,22 +668,33 @@
      ("openssl"). The libraries will be built in the top-level directory,
      and the binary will be in the "apps" subdirectory.
 
+     Troubleshooting hints:
+
      If the build fails, look at the output.  There may be reasons
      for the failure that aren't problems in OpenSSL itself (like
-     missing standard headers).  If you are having problems you can
-     get help by sending an email to the openssl-users email list (see
+     missing standard headers).
+
+     Your best bet to fix build or test failures is to clean the build
```

Comment:
IMHO "Your best bet to build or test failures" sounds a bit too general. How about rewording it to something like that?

> If the build succeeded previously, but fails after a source update, it might be helpful to clean the build tree before attempting another build build.

Reply:
ups, only one 'build' at the end.